app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@tenderlove 

wow - never heard of `run_callbacks`.  Would that set the errors variable that they UI can check?

Other question is your method `nope` ends in `throw :abort`.  I have seen other samples on the web do `return false` instead.  Which is the preferred way?

Final question is does ActiveRecord's destroy method raise errors that the caller needs to catch? Or is the `destroyed?` check adequate?